ts/nni_manager/training_service/v3/compat.ts

```diff
@@ -56,21 +70,29 @@ export class V3asV1 implements TrainingService {
     public async submitTrialJob(form: TrialJobApplicationForm): Promise<TrialJobDetail> {
```

Comment:
The logic of `submitTrialJob` does not keep all the features of the previous version. For example, in the previous version, `form` could be `id` in it, if there is, `trialId` should respect that `id`.